src/app/components/core/configuration-manager/configuration-manager.component.html

```diff
@@ -41,21 +41,34 @@
               <div class="field">
                 <div class="control">
                   <p hidden="true"> {{v.value.type}} </p>
-                  <input *ngIf='isObject(v?.value?.value)' id="{{category.key.trim() + '-' + v.key.trim() | lowercase }}" class="input is-small" type="text"
-                    [value]="v?.value?.value != undefined ? (JSON.stringify(v?.value?.value)) : '' " [textContent]="v?.value?.value != undefined ? (JSON.stringify(v?.value?.value)) : '' "
-                    (input)="onTextChange(category.key.trim() + '-' + v.key.trim())" />
-
-                  <input *ngIf='!isObject(v?.value?.value)' id="{{category.key.trim() + '-' + v.key.trim() | lowercase }}" class="input is-small" type="text"
-                    [value]='v?.value?.value != undefined ? v?.value?.value : "" ' [textContent]='v?.value?.value != undefined ? v?.value?.value : "" '
-                    (input)="onTextChange(category.key.trim() + '-' + v.key.trim())" />
+                  <div class="control is-fullwidth">
+                    <form name="form" id="ngForm" #f="ngForm" novalidate>
+                      <select *ngIf="(v.value.type | uppercase) == 'BOOLEAN'" id="{{category.key.trim() + '-' + v.key.trim() | lowercase }}" (input)="onTextChange(category.key.trim() + '-' + v.key.trim())"
```

Comment:
it looks ugly! 

Please use `<div class="select" *ngIf="(v.value.type | uppercase) == 'BOOLEAN'">` and see why Purge -> retainUnsent is not binding! Looks it has value True/ False while expected model binding is true/false!